src/backend/src/services/design-review.services.ts

```diff
@@ -45,6 +52,91 @@ export default class DesignReviewService {
     return designReviewTransformer(deletedDesignReview);
   }
 
+  /**
+   *
+   * @param user user who created the design review
+   * @param dateScheduled when the design review is scheduled for
+   * @param teamTypeId team type id the design review concerns
+   * @param requiredMembers users who need to be at the design review
+   * @param optionalMembers users who do not need to be at the design review
+   * @param location where the design review will be held
+   * @param isOnline if the design reivew is held online
+   * @param isInPerson if the design review is held in person
+   * @param zoomLink what the zoom link is
+   * @param docTemplateLink doc template link
+   * @param wbsElementId wbs element id the design review is related to
+   * @param meetingTimes meeting times
+   * @throws if permissions are not valid to create a design review
+   * @returns a newly created design review
+   */
+  static async createDesignReview(
+    user: User,
+    dateScheduled: string,
+    teamTypeId: string,
+    requiredMembers: User[],
+    optionalMembers: User[],
+    location: string,
+    isOnline: boolean,
+    isInPerson: boolean,
+    zoomLink: string,
+    docTemplateLink: string,
+    wbsElementId: number,
+    meetingTimes: number[]
+  ): Promise<Design_Review> {
+    if (isGuest(user.role)) throw new AccessDeniedGuestException('create design review');
+
+    const teamType = await prisma.teamType.findFirst({
+      where: { teamTypeId }
+    });
+
+    if (!teamType) {
+      throw new NotFoundException('Team Type', teamTypeId);
+    }
+
+    const WBS_Element = await prisma.wBS_Element.findUnique({
+      where: { wbsElementId }
+    });
+
+    if (!WBS_Element) {
+      throw new NotFoundException('WBS Element', wbsElementId);
+    }
+
```

Comment:
you should also check that the set of meetingTimes is valid (should be in a valid range, and be contiguous, e.g. 1, 2, 3 is valid, 1, 3 is not)